common/src/main/scala/vidispine/VSLazyItem.scala

```diff
@@ -69,6 +79,7 @@ case class VSLazyItem (itemId:String, lookedUpMetadata:Map[String,VSMetadataEntr
 }
 
 object VSLazyItem extends ((String,Map[String,VSMetadataEntry],Option[Map[String,VSShape]])=>VSLazyItem) {
+  private val logger = LoggerFactory.getLogger(getClass)
```

Comment:
can take this out again if nothing is using it

Reply:
Done.

Reply:
Removing this broke something so I put it back.